.travis.yml

```diff
@@ -20,6 +20,9 @@ install:
 before_script:
   - nix-shell _apps/data/shell.nix --pure --command "echo 'run nix-shell and exit'"
 script:
+  - cd _data/python-pfhub
+  - nix-shell --command "cd ../../results && make notebooks"
+  - cd ../..
   - nix-shell --command "
```

Comment:
```suggestion
  - nix-shell --command "cd results && make notebooks"
  - nix-shell --command "
```

Reply:
The [shell.nix](https://github.com/usnistgov/pfhub/blob/master/_data/python-pfhub/shell.nix) file needs to be launched from the directory with the python package in it if it is used as a development package. This is the shell.nix file in `_data/python-pfhub`. Using as a development package means that edited python files can be immediately be used in the notebooks without rerunning the nix install. This is a weird interaction with setuptools and nix installing from the nix-shell. If pfhub is listed as a package in the nix environment than this wouldn't be the case, but then it isn't a dev package and all it's files are stored in the nix-store and need to be updated everytime I make a change the python-pfhub and then run a notebook.